src/components/Dialog/BaseDialog/BaseDialog.tsx

```diff
@@ -50,10 +55,13 @@ export const BaseDialog: FC<BaseDialogProps> = React.forwardRef(
             styles.dialogBackdrop,
             dialogWrapperClassNames,
             { [styles.visible]: visible },
+            { [styles.modeless]: overlay === false },
+            { [styles.modelessMask]: overlay === false && maskClosable },
```

Comment:
this can be changed into one single object

Reply:
unfortunately, maskClosable is already part of the API. need to use 'as is'.

Reply:
We should deprecate using a decorator, but not remove the prop until the next major release.

Reply:
My comment was to move all the styles into a single object.
```suggestion
            { [styles.modeless]: overlay === false,
              [
              styles.modelessMask]: overlay === false && maskClosable },
```